Nicholas Mather/week01/word_guesser/js/exercises.js

```diff
@@ -0,0 +1,77 @@
+// # Homework: The Word Guesser
+//
+// You'll create a simple word guessing game where the user gets infinite tries to guess the word (like Hangman without the hangman, or like Wheel of Fortune without the wheel and fortune).
+//
+// - Create two global arrays: one to hold the letters of the word (e.g. 'F', 'O', 'X'), and one to hold the current guessed letters (e.g. it would start with '\_', '\_', '\_' and end with 'F', 'O', 'X').
+
+const targetWord = ["r", "e", "d"];
+const guessedLetters = ['_','_','_'];
+let newLetter;
+let numRight = targetWord.length;
+// - Write a function called guessLetter that will:
+// - Take one argument, the guessed letter.
+// - Iterate through the word letters and see if the guessed letter is in there.
+// - If the guessed letter matches a word letter, changed the guessed letters array to reflect that.
+// - When it's done iterating, it should log the current guessed letters ('F__')
+// and congratulate the user if they found a new letter.
+// - It should also figure out if there are any more letters that need to be guessed,
+// and if not, it should congratulate the user for winning the game.
+// - Pretend you don't know the word, and call guessLetter multiple times with various letters to check that your program works.
+
+const guessLetter = function(guess) {
+  for (i = 0; i < targetWord.length; i++) {
+    if (guess === targetWord[i]) {
+      guessedLetters[i] = guess; //add correct guess to guessedLetters
```

Comment:
Need to use the .push() method here. 
i.e. `guessedLetters.push(guess)`
This will add the guess letter into the guessedLetters array